index.bs

```diff
@@ -0,0 +1,158 @@
+<pre class='metadata'>
+Title: Runtime Keys
+Shortname: runtime-keys
+Level: 1
+Status: w3c/CG-DRAFT
+Group: wintercg
+URL: http://runtime-keys.proposal.wintercg.org
+Editor: Ethan Arrowood, Vercel http://vercel.com, ethan-arrowood@vercel.com
+Abstract: A proposal defining standard identifier keys for various runtime environments
+Markup Shorthands: markdown yes
+</pre>
+<pre class=link-defaults>
+spec:url; type:interface; text:URL
+spec:html; type:attribute; for:Window; text:navigator
+spec:html; type:attribute; for:Window; text:self
+</pre>
+
+Introduction {#intro}
+=====================
+
+This proposal defines a list of keys that represent various runtime environments. These keys can be used in a variety of ways, and should be considered a reliable representation of the given runtime. For example, they may be used in a project configuration file to indicate the given project supports the specified runtime. This specification does not detail <i>how</i> the keys can be used; that is left up to community developed tools. The point of this proposal is too only define <i>what</i> the keys are in order to prevent conflicts and give users a reliable list of platforms they can build tooling around.
+
+Keys {#keys}
+============
+
+## Cloudflare - workerd ## {#workerd}
+
+The JavaScript / Wasm runtime that powers Cloudflare Workers.
+
+Key: `workerd`
+
+Website: [https://workers.cloudflare.com/](https://workers.cloudflare.com/)
+
+Repository: [https://github.com/cloudflare/workerd](https://github.com/cloudflare/workerd)
+
+## Deno Land - Deno ## {#deno}
+
+A modern runtime for JavaScript and TypeScript.
+
+Key: `deno`
+
+Website: [https://deno.com](https://deno.com)
+
+Repository: [https://github.com/denoland/deno](https://github.com/denoland/deno)
+
+## Meta - React Native ## {#react-native}
+
+Create native apps for Android and iOS using React.
+
+Key: `react-native`
```

Comment:
Feedback from the React Native team:

```suggestion
## React Native ## {#react-native}

A framework for building native apps using React. This key represents supported React Native JS runtimes on native platforms (excludes react-native-web).

Key: `react-native`
```
- We'd encourage dropping the "Meta" prefix here and going simply with "React Native".
- `react-native` (as a runtime) conceptually makes sense. Please see comments on https://github.com/react-native-community/discussions-and-proposals/discussions/535#discussioncomment-3925106 for more info (and **thank you** for sharing this there).

Reply:
As you can see with the rest of the list, every project has its relative organization or company in front of it. Is there another entity we can use there? During our call today I'll ask the group if having these organizations in front of the runtimes is necessary and we'll go from there.

Reply:
Got it — if aiming for consistency this is fine. Meta is the correct entity 👍.